config.toml

```diff
@@ -221,7 +222,7 @@ googleAnalytics = "UA-60791139-1"
 			name = "SoCraTes Conf Switzerland"
 			position = "Switzerland"
 			social = [
-				["fa-twitter", "https://twitter.com/SoCraTes_CH"],
+				["fab fa-mastodon", "https://tooting.ch/@socrates_ch"],
```

Comment:
I know that this is the link used in the official SoCraTes CH website, but i have a not found page